focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
```

Comment:
Yeah, to me it is intuitive to take the points from the center, especially considering we have to map these by hand. I'll add the comments accordingly.